HyperV/templates/cinder.conf

```diff
@@ -3,7 +3,7 @@ auth_strategy=keystone
 volume_name_template=volume-%s
 volume_driver=${volumeDriver}
 glance_api_servers=${devstackIp}:9292
-sql_connection=mysql://root:Passw0rd@${devstackIp}/cinder
+sql_connection=mysql+pymysql://root:Passw0rd@${devstackIp}/cinder
```

Comment:
In the first place, why do we have this cinder.conf template on the nova CI? We're not using the Windows Cinder drivers in this case, but the default LVM driver instead.